core/src/main/java/io/fabric8/maven/core/util/kubernetes/KubernetesClientUtil.java

```diff
@@ -225,4 +231,68 @@ protected static String getPodCondition(Pod pod) {
         return "";
     }
 
+    public static Map<String, Object> doCreateCustomResource(KubernetesClient kubernetesClient, CustomResourceDefinitionContext crdContext, String namespace, File customResourceFile) throws IOException {
+        if (crdContext.getScope().equals("Namespaced")) {
```

Comment:
Do `Namepsaced`.equals(crdContext....)` and same in the other statements of this class.